app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
@tijmenb Oh I see what you mean. I guess all types of unpublishing will have some kind of record in the live content store - @gpeng could confirm.